src/DssSpell.t.sol

```diff
@@ -659,139 +666,72 @@ contract DssSpellTest is DSTest, DSMath {
+        //assertEq(pipGUSD.bud(address(end)), 1);
+        //assertEq(MedianAbstract(pipGUSD.src()).bud(address(pipGUSD)), 1);
 
         // Join to adapter
-        assertEq(vat.gem("YFI-A", address(this)), 0);
-        yfi.approve(address(joinYFIA), ilkAmt);
-        joinYFIA.join(address(this), ilkAmt);
-        assertEq(yfi.balanceOf(address(this)), 0);
-        assertEq(vat.gem("YFI-A", address(this)), ilkAmt);
+        assertEq(vat.gem("GUSD-A", address(this)), 0);
+        gusd.approve(address(joinGUSDA), ilkAmt);
+        joinGUSDA.join(address(this), ilkAmt);
+        assertEq(gusd.balanceOf(address(this)), 0);
+        assertEq(vat.gem("GUSD-A", address(this)), ilkAmt * 10**16);
 
         // Deposit collateral, generate DAI
         assertEq(vat.dai(address(this)), 0);
-        vat.frob("YFI-A", address(this), address(this), address(this), int(ilkAmt), int(100 * WAD));
-        assertEq(vat.gem("YFI-A", address(this)), 0);
+        vat.frob("GUSD-A", address(this), address(this), address(this), int(ilkAmt * 10**16), int(100 * WAD));
+        assertEq(vat.gem("GUSD-A", address(this)), 0);
         assertEq(vat.dai(address(this)), 100 * RAD);
 
         // Payback DAI, withdraw collateral
-        vat.frob("YFI-A", address(this), address(this), address(this), -int(ilkAmt), -int(100 * WAD));
-        assertEq(vat.gem("YFI-A", address(this)), ilkAmt);
+        vat.frob("GUSD-A", address(this), address(this), address(this), -int(ilkAmt * 10**16), -int(100 * WAD));
+        assertEq(vat.gem("GUSD-A", address(this)), ilkAmt * 10**16);
         assertEq(vat.dai(address(this)), 0);
 
         // Withdraw from adapter
-        joinYFIA.exit(address(this), ilkAmt);
-        assertEq(yfi.balanceOf(address(this)), ilkAmt);
-        assertEq(vat.gem("YFI-A", address(this)), 0);
+        joinGUSDA.exit(address(this), ilkAmt);
+        assertEq(gusd.balanceOf(address(this)), ilkAmt);
+        assertEq(vat.gem("GUSD-A", address(this)), 0);
 
         // Generate new DAI to force a liquidation
-        yfi.approve(address(joinYFIA), ilkAmt);
-        joinYFIA.join(address(this), ilkAmt);
-        (,,uint256 spotV,,) = vat.ilks("YFI-A");
+        gusd.approve(address(joinGUSDA), ilkAmt);
+        joinGUSDA.join(address(this), ilkAmt);
+        (,,uint256 spotV,,) = vat.ilks("GUSD-A");
         // dart max amount of DAI
-        vat.frob("YFI-A", address(this), address(this), address(this), int(ilkAmt), int(mul(ilkAmt, spotV) / RAY));
+        vat.frob("GUSD-A", address(this), address(this), address(this), int(ilkAmt * 10**16), int(mul(ilkAmt * 10**16, spotV) / RAY));
         hevm.warp(now + 1);
-        jug.drip("YFI-A");
-        assertEq(flipYFIA.kicks(), 0);
-        cat.bite("YFI-A", address(this));
-        assertEq(flipYFIA.kicks(), 1);
+        jug.drip("GUSD-A");
+        assertEq(flipGUSDA.kicks(), 0);
+        //cat.bite("GUSD-A", address(this));  // liquidatons off
+        //assertEq(flipGUSDA.kicks(), 1);
     }
```

Comment:
For completeness, should add an assertion for the `FlipFab` chainlog entry.

Reply:
done